inspire/modules/workflows/tasks/matching.py

```diff
@@ -248,4 +248,5 @@ def _update_old_object(obj, eng):
 def arxiv_set_category_field(obj, eng):
     """Temporary measure to enable sorting by primary category."""
     record = get_record_from_obj(obj, eng)
-    obj.uri = record.get("report_number.arxiv_category", [""])[0]
+    obj.uri = record['arxiv_eprints'][0]['categories'][0]
```

Comment:
Hey Jan ,
I tried it and it was failing the test, so thats why I changed and i left
the comment so as to check it together. I will double check tomorrow and I
will inform you. Cheers!
Στις 28 Σεπ 2015 5:57 μ.μ., ο χρήστης "Jan Åge Lavik" <
notifications@github.com> έγραψε:

> In inspire/modules/workflows/tasks/matching.py
> <https://github.com/inspirehep/inspire-next/pull/362#discussion_r40569265>
> :
>
> > @@ -248,4 +248,5 @@ def _update_old_object(obj, eng):
> >  def arxiv_set_category_field(obj, eng):
> >      """Temporary measure to enable sorting by primary category."""
> >      record = get_record_from_obj(obj, eng)
> > -    obj.uri = record.get("report_number.arxiv_category", [""])[0]
> > +    obj.uri = record['arxiv_eprints'][0]['categories'][0]
>
> I think this could still be accessed like record.get("arxiv_eprints.categories".
> [""])[0] (as record is a Record object at this point. If you can try it
> it would be great. Then you can also remove the commented line below.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/362/files#r40569265>.
>